server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -77,6 +78,11 @@ const accountMutations = {
           throwIfMissing: true,
         });
 
+        // Enforce 2FA if trying to change 2FA rolling limit settings while it's already enabled
+        if (args.key.split('.')[0] === 'payoutsTwoFactorAuth' && account.settings?.payoutsTwoFactorAuth?.enabled) {
+          await TwoFactorAuthLib.validateRequest(req, { alwaysAskForToken: true });
```

Comment:
@Betree 

```suggestion
          await TwoFactorAuthLib.validateRequest(req, { alwaysAskForToken: true, requireTwoFactorAuthEnabled: true });
```

`alwaysAskForToken` will require a token if the user has 2FA enabled, otherwise the check will pass
`requireTwoFactorAuthEnabled` will make sure that the user also has 2FA enabled.